File: articles/strategic-robustness-across-futures/fortran/robustness_score.f90
program robustness_score
  implicit none
  real :: values(4)
  real :: worst_case

  values = (/0.66, 0.72, 0.78, 0.70/)
  worst_case = minval(values)

  print *, "Robust resilience portfolio worst-case viability:", worst_case
end program robustness_score

File: articles/strategic-robustness-across-futures/cpp/robustness_score.cpp
#include <algorithm>
#include <iostream>
#include <vector>

double worst_case_viability(const std::vector<double>& values) {
    return *std::min_element(values.begin(), values.end());
}

int main() {
    std::vector<double> robust_resilience = {0.66, 0.72, 0.78, 0.70};
    std::cout << "Robust resilience portfolio worst-case viability="
              << worst_case_viability(robust_resilience) << "\n";
    return 0;
}
